### ISSUE TITLE
Remove "Send Results" Message

### ISSUE THREAD

**Issue by theplacer** [SpigotMC/BungeeCord#3014]
Hello, I noticed that whenever I send a player or multiple players to a specific server, I get this send results message that is multiple lines long. It gets kind of spammy in the chat. Is there any way to remove this?

![Screen Shot 2020-11-02 at 1 49 10 PM](https://user-images.githubusercontent.com/72987400/104047832-e8ad6c80-51a7-11eb-8b28-b6844fc1a248.png)

**Comment by Mystiflow** [comment]
Yeah I agree that is rather unpleasant to look at. I made it more compact; if this looks good to you, I can PR once I get my SSH and repo paths set up
![Example](https://user-images.githubusercontent.com/10323494/104056094-cf2c1500-51e7-11eb-8ca6-b7041f16290e.png)

**Comment by theplacer** [comment]
Looks good. Can it be light red (&c) and gray (&7) because those are my server colors

**Comment by Mystiflow** [comment]
I put some configuration settings in instead:
```
command_send_no_players=\u00a7cNo players selected to send
command_send_attempting=\u00a72Attempting to send {0} to {1}
command_send_delimiter=\u00a77, 
command_send_results=\u00a7aSend Results: 
command_send_counter=\u00a76({0})
command_send_server_title_color_char=b
```
